syntax = "proto3";

package notify;

message NotifyRequest {
  Item item = 1;
  string ip = 2;
  string port = 3;
}

message Item {
  string itemId = 1;
  float price = 2;
  string name = 3;
  string category = 4;
  string description = 5;
  int32 quantity = 6;
  float rating = 7;
}

message NotifyResponse {
  string message = 1;
}

service NotifyService {
  rpc Notify (NotifyRequest) returns (NotifyResponse);
}